sorted_words.rb

```diff
@@ -5,13 +5,14 @@
 
 words = []
 
-    print "Enter as many words (or press Enter to finish)? "
+loop do
+    print "Enter a word (or press Enter to exit): "
     adding_word = gets.chomp
 
-    while adding_word != ""
-        words.push(adding_word)
-        adding_word = gets.chomp
-        end
+    break if adding_word == ""
+    words.push(adding_word)
+end
+
 
 
 sorted = words.sort
```

Comment:
Nice! It's easier for the user to enter each individual word now and I can see the words sorted in the sorted variable. One small thing, the exercise asks for the sorted words to be printed so I would just add one last line in to print the sorted variable. Other than that, this is good to merge - great job!